### PR TITLE
community section update + add stuff about support

### PR DESCRIPTION
**Pull request by ElPaisano**
# Draft 👉  https://bafybeibiyrgmm6xz4p3lyhhuyklzrqpojhjdni5422nrtkjedcxyunowse.on.fleek.co/
A second refactor of the Projects / Community section and sidebar menu. Addresses:
1. Restructures community page into different groups ->  https://bafybeibiyrgmm6xz4p3lyhhuyklzrqpojhjdni5422nrtkjedcxyunowse.on.fleek.co/community/
1. https://github.com/ipfs/ipfs-docs/issues/1469 adds a section getting dev support ->  https://bafybeibiyrgmm6xz4p3lyhhuyklzrqpojhjdni5422nrtkjedcxyunowse.on.fleek.co/community/#get-technical-support-and-help
1. Removes outdated community links (ProtoSchool isn't really a thing anymore, some repos in the main sidebar are no longer active) -> the Projects section sidebar and  https://bafybeibiyrgmm6xz4p3lyhhuyklzrqpojhjdni5422nrtkjedcxyunowse.on.fleek.co/community/
1. Shuffles sidebar menu items around 
1. Adds notes to a few pages about code of conduct and reporting abuse https://github.com/ipfs/ipfs-docs/pull/1473#issuecomment-1412830962

Goals of this PR:
- Improve navigability
- Improve grouping
- Improve sidebar menu options 
- Remove outdated / items
- Remove as many outbound links from the sidebar menus as possible (bad user experience)
- Add info on getting tech / dev support

## Checklist before requesting a review
- [ ] Passing the beta version of the **Check Markdown links for modified files** check. Action results can be viewed [here](https://github.com/ipfs/ipfs-docs/actions/workflows/action.yml).

## Checklist before merging
- [ ] Passing all required checks (The beta **Check Markdown links for modified files** check is not required)

### CODE REVIEW

**Comment by ElPaisano** [comment]
Was looking through open issues and realized that Task 1 and Task 2 on https://github.com/ipfs/ipfs-docs/issues/1458#issuecomment-1405549949 could also be completed here. So I added note to a few pages RE community code of conduct and reporting abuse
- https://bafybeicdiky2jwx3chhcgsdnmuuqob5tb2a53tjbvmocxveodtxpioek5m.on.fleek.co/project/
- https://bafybeicdiky2jwx3chhcgsdnmuuqob5tb2a53tjbvmocxveodtxpioek5m.on.fleek.co/community/
- https://bafybeicdiky2jwx3chhcgsdnmuuqob5tb2a53tjbvmocxveodtxpioek5m.on.fleek.co/community/contribute/ways-to-contribute/#code

File: docs/community/README.md [hunk 1]
```diff
@@ -1,52 +1,133 @@
...
+- Stay informed and up to date on the latest developments in the IPFS ecosystem with [social media](#social-media) and [the IPFS newsletter](#ipfs-newsletter).
+- Navigate the IPFS ecosystem using [the ecosystem directory](#ecosystem-directory) and [Awesome IPFS](#awesome-ipfs).
 
-No matter your area of specialty or level of expertise, there are many [ways to contribute](contribute/ways-to-contribute.md) to IPFS and make a real difference in the IPFS community at large.
+## Get technical support and help
+
+Are you developing with IPFS implementations and tools, and looking for technical support from IPFS experts? For the fastest possible assistance and resolution of your support needs, do the following:
+
+1. [Check to see if the problem / issue you've encountered has already been discussed](#_1-check-to-see-if-the-issue-you-ve-encountered-has-already-been-discussed).
+2. If the your problem / issue has not been discussed or solved in the forum, [follow the general guidelines for getting useful answers to your questions](#_2-follow-the-guidelines-for-getting-useful-answers-to-your-questions).
```
Comment (2color) [inline docs/community/README.md:26]:
```suggestion
2. If your problem / issue has not been discussed or solved in the forum, [follow the general guidelines for getting useful answers to your questions](#_2-follow-the-guidelines-for-getting-useful-answers-to-your-questions). 
```

File: docs/community/README.md [hunk 2]
```diff
@@ -1,52 +1,133 @@
...
+
+:::callout
+In general, public IPFS Discord channels are bridged to Matrix and Slack. As a result, you can usually use your preferred chat application to engage with the community. IPFS channels/rooms are usually prefixed with "#ipfs-".
+:::
 
-ProtoSchool isn't just home to interactive tutorials on the decentralized web — it's also a vibrant community of makers and doers around the globe! Community groups worldwide host [ProtoSchool workshops](https://proto.school/events) regularly, offering you the opportunity to complete our interactive tutorials with the support of local mentors.
+- **Discord**: Join the [IPFS Discord server](https://discord.gg/ipfs) for synchronous chat! [Server documentation can be found on Notion](https://pl-strflt.notion.site/IPFS-Discord-Server-Documentation-85bbc451303a473bbf6846b01610e3c1).
 
-## Meetups
+- **Matrix**: Join the [Matrix space](https://matrix.to/#/#ipfs-space:ipfs.io). [Home server documentation is available on Notion](https://pl-strflt.notion.site/IPFS-Matrix-Homeserver-Documentation-31481e76843547b7ab5160a87eed2b9f).
```
Comment (2color) [inline docs/community/README.md:91]: I don't think the notion page is public

File: docs/community/README.md [hunk 3]
```diff
@@ -1,52 +1,133 @@
...
+
+- **Slack**: Join [Filecoin slack](https://filecoin.io/slack) and join the bridged #ipfs channels. The [Filecoin Slack workspace user guide is available on Notion](https://pl-strflt.notion.site/Filecoin-Slack-User-Guide-29072679986b4ccb8ad7b091097dd770).
```
Comment (2color) [inline docs/community/README.md:93]: I don't think this notion page is public either

File: docs/community/README.md [hunk 4]
```diff
@@ -1,52 +1,133 @@
...
+### Chat
+
+Casual chat fan? You can engage with the IPFS community in one of the channels listed below:
+
+:::callout
+In general, public IPFS Discord channels are bridged to Matrix and Slack. As a result, you can usually use your preferred chat application to engage with the community. IPFS channels/rooms are usually prefixed with "#ipfs-".
+:::
 
-ProtoSchool isn't just home to interactive tutorials on the decentralized web — it's also a vibrant community of makers and doers around the globe! Community groups worldwide host [ProtoSchool workshops](https://proto.school/events) regularly, offering you the opportunity to complete our interactive tutorials with the support of local mentors.
+- **Discord**: Join the [IPFS Discord server](https://discord.gg/ipfs) for synchronous chat! [Server documentation can be found on Notion](https://pl-strflt.notion.site/IPFS-Discord-Server-Documentation-85bbc451303a473bbf6846b01610e3c1).
```
Comment (2color) [inline docs/community/README.md:89]: Same here — Notion page isn't meant to be public

File: docs/.vuepress/config.js [hunk 5]
```diff
@@ -296,17 +276,12 @@ module.exports = {
             '/project/repository-guide',
             '/project/related-projects',
             '/project/implementation-status',
-            ['https://github.com/ipfs/roadmap', 'Roadmap'],
```
Comment (2color) [inline docs/.vuepress/config.js:299]: Why are we removing the link to the roadmap?

File: docs/.vuepress/config.js [hunk 6]
```diff
@@ -322,14 +297,8 @@ module.exports = {
...
                 '/project/implementation-status',
-                ['https://github.com/ipfs/roadmap', 'Roadmap'],
                 ['https://github.com/ipfs/specs', 'Specifications'],
-                ['https://github.com/ipfs/research', 'Research'],
-                ['https://github.com/ipfs/team-mgmt', 'Team org planning'],
-                [
-                  'https://github.com/ipfs/community/blob/master/code-of-conduct.md',
-                  'Code of conduct'
-                ]
+                ['https://github.com/ipfs/notes', 'Research'],
```
Comment (2color) [inline docs/.vuepress/config.js:301]: I think this should link to the docs research page rather than https://github.com/ipfs/notes

File: docs/.vuepress/config.js [hunk 7]
```diff
@@ -322,14 +297,8 @@ module.exports = {
                 '/project/repository-guide',
                 '/project/related-projects',
                 '/project/implementation-status',
-                ['https://github.com/ipfs/roadmap', 'Roadmap'],
                 ['https://github.com/ipfs/specs', 'Specifications'],
-                ['https://github.com/ipfs/research', 'Research'],
-                ['https://github.com/ipfs/team-mgmt', 'Team org planning'],
-                [
```
Comment (2color) [inline docs/.vuepress/config.js:329]: I would still leave the link to the code of conduct in the sidebar menu.

File: docs/.vuepress/config.js [hunk 8]
```diff
@@ -296,17 +276,12 @@ module.exports = {
...
             '/project/implementation-status',
-            ['https://github.com/ipfs/roadmap', 'Roadmap'],
             ['https://github.com/ipfs/specs', 'Specifications'],
-            ['https://github.com/ipfs/research', 'Research'],
-            ['https://github.com/ipfs/team-mgmt', 'Team org planning'],
-            [
-              'https://github.com/ipfs/community/blob/master/code-of-conduct.md',
-              'Code of conduct'
-            ],
+            ['https://github.com/ipfs/notes', 'Research'],
```
Comment (2color) [inline docs/.vuepress/config.js:280]: Same thing as https://github.com/ipfs/ipfs-docs/pull/1473/files#r1094526073

**Review by 2color — APPROVED** [review]
LGTM.

The main feedback would be to leave the sidebar link to the code of conduct.

File: docs/.vuepress/config.js (hunk 5 above)
Comment (ElPaisano) [reply docs/.vuepress/config.js:299]: It's not clear to me that this repo is still being used. Last commit was end of October. Is this repo no longer being used for roadmap stuff? Aren't roadmaps individualized to a product now?

**Comment by ElPaisano** [comment]
@2color other than the roadmap repo, I think https://github.com/ipfs/ipfs-docs/pull/1473/commits/809e722eecaf6c6a5d49f6c6f947b1a68567f554 addresses the rest of your feedback

**Comment by ElPaisano** [comment]
Confirming that this PR has no spelling issues (Vale is flagging special terms, need to update word list) or broken links

**Comment by ElPaisano** [comment]
@walkerlj0 didn't get a chance to mention this in the working group - this PR (merged) updates https://docs.ipfs.tech/community/

TLDR:
- Adds info on getting tech support
- Rearranges menu items
- Removes outdated items and copy
